lib/ansible/modules/network/avi/avi_analyticsprofile.py

```diff
@@ -30,130 +30,154 @@
+        type: str
     avi_api_update_method:
         description:
             - Default method for object update is HTTP PUT.
             - Setting to patch will override that behavior to use HTTP PATCH.
         version_added: "2.5"
         default: put
         choices: ["put", "patch"]
+        type: str
     avi_api_patch_op:
         description:
             - Patch operation to use when using avi_api_update_method as patch.
         version_added: "2.5"
         choices: ["add", "replace", "delete"]
+        type: str
     apdex_response_threshold:
         description:
             - If a client receives an http response in less than the satisfactory latency threshold, the request is considered satisfied.
             - It is considered tolerated if it is not satisfied and less than tolerated latency factor multiplied by the satisfactory latency threshold.
             - Greater than this number and the client's request is considered frustrated.
             - Allowed values are 1-30000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 500.
+        type: int
     apdex_response_tolerated_factor:
         description:
             - Client tolerated response latency factor.
             - Client must receive a response within this factor times the satisfactory threshold (apdex_response_threshold) to be considered tolerated.
             - Allowed values are 1-1000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 4.0.
+        type: float
     apdex_rtt_threshold:
         description:
             - Satisfactory client to avi round trip time(rtt).
             - Allowed values are 1-2000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 250.
+        type: int
     apdex_rtt_tolerated_factor:
         description:
             - Tolerated client to avi round trip time(rtt) factor.
             - It is a multiple of apdex_rtt_tolerated_factor.
             - Allowed values are 1-1000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 4.0.
+        type: float
     apdex_rum_threshold:
         description:
             - If a client is able to load a page in less than the satisfactory latency threshold, the pageload is considered satisfied.
-            - It is considered tolerated if it is greater than satisfied but less than the tolerated latency multiplied by satisfied latency.
+            - It is considered tolerated if it is greater than satisfied but less than the tolerated latency multiplied by satisifed latency.
             - Greater than this number and the client's request is considered frustrated.
-            - A pageload includes the time for dns lookup, download of all http objects, and page render time.
+            - A PageLoad includes the time for DNS lookup, download of all HTTP objects, and page render time.
```

Comment:
Here too, I wouldn't capitalize PageLoad (though I would capitalize DNS and HTTP).
```suggestion
            - A pageload includes the time for DNS lookup, download of all HTTP objects, and page render time.
```